client/src/utilities/auth.js

```diff
@@ -1,100 +1,97 @@
 /* eslint-disable no-underscore-dangle */
 import Keycloak from 'keycloak-js';
-import Raven from 'raven-js';
-import queryString from 'querystringify';
-
+import Auth0 from 'auth0-js';
+import url from 'url';
+import * as k from './keycloak';
+import * as a0 from './auth0';
 import { get } from './api';
 
 let keycloak = null;
+let auth0 = null;
```

Comment:
Do we need two var `keycloak` and `auth0`? Maybe one `auth` is enough and simplifies the logic below? 

Or maybe we have to do the OO way so that the logout fn looks like:

```
export function logout() {
	  authObject.logout();
}
```

Check with Charles or Martin about how to properly do it.

Reply:
I agree that the solution is not the best but as far as the solution is a provisional trick to have 2 authentication systems, is wrapped inside one file `auth.js` and should be removed shortly ....

could we go ahead with it and spend the energy in other more long term parts/issues/codes?

Reply:
IMO, should be more practical to have the PR in `test` so we could test deeply the code